Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:11.4 as base
+FROM debian:11.5 as base
 
-LABEL maintainers="pastakhov@yandex.ru,alexey@wikiteq.com"
+LABEL maintainers="pavel@wikiteq.com,alexey@wikiteq.com"
```

Comment:
I suggest splitting up this patch to clarify what is the 1.39.1 stuff and what is separate - updating the email can be done right away without waiting for the rest of the patch